packages/notification-spa/set-env.ts

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+
+const writeFile = fs.writeFile;
+
+const targetPath = './src/environments/environment.ts';
+
+require('dotenv').config();
+
+const envConfigFile = `export const environment = {
+   API_URL: '${process.env.API_URL}',
+   production: true,
+};
+`;
+console.log('The file `environment.ts` will be written with the following content: \n');
+console.log(envConfigFile);
```

Comment:
```suggestion
```

Reply:
This won't be logged on browser console. This would be run directly on node, it will only print in the CLI